tox.ini

```diff
@@ -25,13 +25,15 @@ deps =
      -r{toxinidir}/requirements_test.txt
      -c{toxinidir}/homeassistant/package_constraints.txt
 commands =
+     python script/gen_requirements_all.py
      pylint homeassistant
 
 [testenv:lint]
 basepython = python3
 deps =
      -r{toxinidir}/requirements_test.txt
 commands =
+         python script/gen_requirements_all.py
```

Comment:
Revert this

You should not modify tox.ini.
Run `python script/gen_requirements_all.py` this will modify the requirements_all.txt file and you commit the changes.